operations/dd-toc-guide.md

```diff
@@ -2,6 +2,12 @@
 
 This document provides the TOC with guidance on how to execute due diligence of CNCF projects for each level of maturity.  It complements the Moving Levels process detailed in the [Process Directory](../process/README.md).
 
+## Quick Links
```

Comment:
rendering issue, for example with `Finalizing DD`

Reply:
should be fixed now. thanks for catching!